lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -108,6 +108,7 @@
     return Date.now();
   }
   helloDate();
+  const d = Date.now(); // FAIL
   eval('Date.now()'); // FAIL
```

Comment:
given our fail/no fail smoke tests right now, what benefit does this add? (seems like we'd be unable to catch `eval` not working here too actually)